requirements_dev.txt

```
@@ -1,13 +1,20 @@
 pytest
 flake8
 pytest-flake8
+ipython
+pytest-notebook
+nbsphinx
+nbval
+nbconvert
 sphinx>=1.7
 bump2version
```

As of May 2020, the bump2version fork has adopted bumpversion to continue development.
```suggestion
bumpversion
```